rustbus/src/connection.rs

```diff
@@ -53,27 +51,9 @@ pub enum Error {
     ConnectionClosed,
 }
 
-impl std::convert::From<std::io::Error> for Error {
-    fn from(e: std::io::Error) -> Error {
-        Error::IoError(e)
-    }
-}
-
-impl std::convert::From<crate::wire::errors::UnmarshalError> for Error {
-    fn from(e: crate::wire::errors::UnmarshalError) -> Error {
-        Error::UnmarshalError(e)
-    }
-}
-
 impl std::convert::From<nix::Error> for Error {
     fn from(e: nix::Error) -> Error {
```

Comment:
Technically this is still `nix` leaking into the public API. No such thing as private impls :(

Reply:
It shouldn't break in too many places to remove this single `From` impl and make the manual conversion right? It's not super elegant but it would actually remove nix from the public API

Reply:
Yep, done.